plinth/__main__.py

```diff
@@ -349,10 +356,13 @@ def main():
         list_dependencies(arguments.list_dependencies)
 
     if arguments.list_modules is not False:
-        list_modules()
+        list_modules(None)
```

Comment:
I think instead of None, you can pass arguments.list_modules to list_modules here.
Then if you run:
- "plinth --list-modules" -> arguments.list_modules will be None.
- "plinth --list-modules essential" -> arguments.list_modules will be "essential".
- "plinth --list-modules optional" -> arguments.list_modules will be "optional".